garage/tf/regressors/gaussian_mlp_regressor.py

```diff
@@ -68,10 +69,14 @@ def __init__(self,
         with tf.variable_scope(name):
 
             if optimizer is None:
+                if optimizer_args is None:
+                    optimizer_args = dict()
                 if use_trust_region:
-                    optimizer = PenaltyLbfgsOptimizer("optimizer")
+                    optimizer = PenaltyLbfgsOptimizer(**optimizer_args)
                 else:
-                    optimizer = LbfgsOptimizer("optimizer")
+                    optimizer = LbfgsOptimizer(**optimizer_args)
+            else:
+                optimizer = optimizer(**optimizer_args)
```

Comment:
Is optimizer_args different from None in this else branch?

Reply:
No. Why?

Reply:
I saw you added this [guard](https://github.com/rlworkgroup/garage/blob/be570838f2fe4c5f291b10247209d04548565370/garage/tf/regressors/gaussian_mlp_regressor.py#L72-L73) for null args, so I was wondering if the same could happen when optimizer is already set, but if that's not the case, it's fine.